tests/test_pyrdl.py

```diff
@@ -130,6 +146,33 @@ def test_any_attr_verify():
     any_constraint.verify(IntData(0), ConstraintContext())
 
 
+def test_anyof_verify():
```

Comment:
why move these?

Reply:
I was moving the `LessThan` and `GreaterThan` closer to where they are now actually used but there's no point doing this, I'll revert